pkg/config/config_test.go

```diff
@@ -141,6 +141,28 @@ clientConnection:
 		t.Fatal(err)
 	}
 
+	metricsCertConfig := filepath.Join(tmpDir, "metrics-cert.yaml")
```

Comment:
Can we have a test where internalCertManagement is false and you specify a different value for certDir?

Reply:
+1 added a `metrics cert config with external certs` test